libs/agno/agno/agent/agent.py

```diff
@@ -850,16 +850,13 @@ def _initialize_session_state(
     def _run(
         self,
         run_response: RunOutput,
+        run_context: RunContext,
```

Comment:
Reminder to just make sure this doesn't break the auto-instrumentor. I don't think it does though, I don't think we use those attributes